app/assets/javascripts/services/settingsSvc.js

```diff
@@ -91,7 +90,39 @@ angular.module('QuepidApp')
           searchEngine:      'os',
           searchUrl:         'https://reader:reader@quepid-opensearch.dev.o19s.com:9000/tmdb/_search',
           urlFormat:         'http(s?)://yourdomain.com:9200/<index>/_search',
+        },
+        vectara: {
+          queryParams:  [
+            '{',
+            '  "query": [',
+            '     {',
+            '       "query": "#$query##",',
+            '       "start": 0,',
+            '       "numResults": 10,',
+            '       "corpusKey": [{',
+            '          "customerId": 123456789,',
+            '          "corpusId": 1,',
+            '          "lexicalInterpolationConfig": {',
+            '            "lambda": 0.025',
+            '          },',
+            '          "dim": []',
+            '       }]',
+            '     }',
+            '  ]',
+            '}'
+          ].join('\n'),
+
+          escapeQuery:       true,
+          apiMethod:         'POST',
           customHeaders:     '',
+          fieldSpec:         'id:id',
+          idField:           'id',
+          titleField:        'title',
```

Comment:
the titleField and idField...  are those values "standard" in Vectara land?   Do we/they want to host a demo Vectara that we can document and use in the wizard, kind of like the TMDB dataset?    And add to the https://github.com/o19s/quepid/wiki/Troubleshooting-Opensearch-and-Quepid#demo-opensearch-indexes-with-the-tmdb-dataset page....